spec/factories/votes.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :vote do
+    user_id { 1 }
```

Comment:
Usually, you'll want this to be an actual associated user (rather than just an integer). See ways to do that here: https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations